packages/integrations/package.json

```diff
@@ -22,6 +22,7 @@
     "tslib": "^1.9.3"
   },
   "devDependencies": {
+    "@sentry/browser": "7.36.0",
```

Comment:
This relationship between these packages was missing and needed to be added to ensure task pipelines know what to do and in what order